core/controllers/topic_editor.py

```diff
@@ -289,12 +265,10 @@ def put(self, topic_id):
             else:
                 topic_and_subtopic_page_change_list.append(
                     topic_domain.TopicChange(change))
-        try:
```

Comment:
This should not be removed. It is required to guard against invalid inputs by raising Invalid Input error to end user.

Reply:
This is linked to [this](https://github.com/oppia/oppia/pull/6750#discussion_r285385782). The exception from this block can't be executed as if an error does occur it will go in the subsequent function and raise the exception from there

Reply:
Hmm... but I thought this was called when an  invalid input was sent. Did you manually test the working with these lines removed?

Reply:
Yes I did. The invalid version is checked from _require_valid_version, the change dicts are checked in their respective domain classes SubtopicPageChange and TopicChange. The commit message is checked in update_topic_and_subtopic_pages

Reply:
Similar comment here to the one in story_domain. Did you try sending a valid changelist that sets an invalid value to a topic field? Like let's say set topic description to a number. Those validation errors are those that raise `utils.ValidationError`.

Reply:
This isn't addressed.

Reply:
Done